src/js/tracing/integrations/appStart.ts

```diff
@@ -123,11 +123,10 @@ export const appStartIntegration = ({
     }
   };
 
-  const afterAllSetup = (client: Client): void => {
+  const afterAllSetup = (_client: Client): void => {
     if (standaloneUserOption === undefined) {
       // If not user defined, set based on the routing instrumentation presence
-      standalone = !client.getIntegrationByName<ReactNativeTracing>(ReactNativeTracing.id)?.options
-        .routingInstrumentation;
+      // FIXME: standalone = getReactNativeTracingIntegration(client)?.options.routingInstrumentation;
```

Comment:
If we are not going to fix it right away, we should at least add a `todo` so it doesn't get lost.

```suggestion
      // TODO: FIXME: standalone = getReactNativeTracingIntegration(client)?.options.routingInstrumentation;
```